swift_browser_ui_frontend/package.json

```diff
@@ -44,7 +44,7 @@
     "babel-eslint": "^10.1.0",
     "babel-jest": "^27.5.1",
     "cli-highlight": "^2.1.11",
-    "csc-ui": "^0.6.4",
+    "csc-ui": "^0.6.5",
```

Comment:
could have switched to 0.6.6

Reply:
Might have come up after Hang moved to `0.6.4`. Dependabot should update it soon.